java/build.gradle.kts

```diff
@@ -28,6 +28,7 @@ import io.spine.internal.dependency.JavaPoet
 import io.spine.internal.dependency.Kotest
 import io.spine.internal.dependency.ProtoData
 import io.spine.internal.dependency.Roaster
+import io.spine.internal.dependency.Spine
```

Comment:
This is weird. Nothing has changed in the script itself. but the import. Please review.